src/main/scala/org/camunda/feel/impl/builtin/NumericBuiltinFunctions.scala

```diff
@@ -100,7 +101,7 @@ object NumericBuiltinFunctions {
 
   private def oddFunction =
     builtinFunction(params = List("number"), invoke = {
-      case List(ValNumber(n)) => ValBoolean(n % 2 == 1)
+      case List(ValNumber(n)) => ValBoolean(abs(n) % 2 == 1)
```

Comment:
We can use the function `abs` directly on the `BigDecimal`. 

```suggestion
      case List(ValNumber(n)) => ValBoolean(n.abs % 2 == 1)
```

Reply:
Done. I did actually see that you also had a similar usages of `.abs` in the absFunction, so make sense.